src/network/clientpackethandler.cpp

```diff
@@ -132,6 +138,8 @@ void Client::handleCommand_Hello(NetworkPacket* pkt)
 
 void Client::handleCommand_AuthAccept(NetworkPacket* pkt)
 {
+	CHECK_RELIABLE(pkt);
```

Comment:
can we have some of that [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)

more seriously: this would be well suited for clientopcodes.cpp and having the check in a central place

Reply:
I considered that but it seemed possible someone might want to send some of these packets as unreliable eventually?

Reply:
separate entry yes